main.py

```diff
@@ -0,0 +1,87 @@
+import random
+import os
+from art import logo
+
+# function calculateScore takes in a list parameter and returns sum of list
+def calculateScore(list):
+    count = sum(list)
+
+    if count == 21:
+        return 21
+    elif count > 21:
+        for num in list:
+            if num == 11:
+                list.remove(num)
+                list.append(1)
+                count = sum(list)
+    
+    return count
+
+# function compareScrore takes in user and dealer's scores, compares them, and returns a string
+def compareScore(userScore, dealerScore):
+  if userScore > 21 and dealerScore > 21:
+    return "You went over. You lose 😤"
+
+  if userScore == dealerScore:
+    return "Draw 🙃"
+  elif dealerScore == 0:
```

Comment:
You're checking for 0 to determine if the dealer won?

Reply:
originally had 0 set as the value for Blackjack. I ended up just going with 21 for less confusion. This was an oversight on my part